docs/ai/tutorials/snippets/evaluate-safety/MyTests.cs

```diff
@@ -0,0 +1,231 @@
+﻿// <SnippetUsingDirectives>
+using Azure.AI.OpenAI;
+using Azure.Identity;
+using Microsoft.Extensions.AI;
+using Microsoft.Extensions.AI.Evaluation;
+using Microsoft.Extensions.AI.Evaluation.Reporting;
+using Microsoft.Extensions.AI.Evaluation.Reporting.Storage;
+using Microsoft.Extensions.AI.Evaluation.Safety;
+using Microsoft.Extensions.Configuration;
+// </SnippetUsingDirectives>
+
+namespace MyTestsNS;
+
+[TestClass]
+public sealed class MyTests
+{
+    // <SnippetScenarioName>
+    private string ScenarioName =>
+        $"{TestContext!.FullyQualifiedTestClassName}.{TestContext.TestName}";
+    private static string ExecutionName =>
+        $"{DateTime.Now:yyyyMMddTHHmmss}";
+    // </SnippetScenarioName>
+
+    // <SnippetTestContext>
+    // The value of the TestContext property is populated by MSTest.
+    public TestContext? TestContext { get; set; }
+    // </SnippetTestContext>
+
+    // <SnippetGetEvaluators>
+    private static IEnumerable<IEvaluator> GetSafetyEvaluators()
+    {
+        IEvaluator violenceEvaluator = new ViolenceEvaluator();
+        yield return violenceEvaluator;
+
+        IEvaluator hateAndUnfairnessEvaluator = new HateAndUnfairnessEvaluator();
+        yield return hateAndUnfairnessEvaluator;
+
+        IEvaluator protectedMaterialEvaluator = new ProtectedMaterialEvaluator();
+        yield return protectedMaterialEvaluator;
+
+        IEvaluator indirectAttackEvaluator = new IndirectAttackEvaluator();
+        yield return indirectAttackEvaluator;
+    }
+    // </SnippetGetEvaluators>
+
+    // <SnippetGetTags>
+    private static IEnumerable<string> GetTags(string storageKind = "Disk")
```

Comment:
Tags are optional and somewhat orthogonal to the main topic of the tutorial (i.e. safety evals) and this is one thing we could omit to reduce the size of the code. That said, we have not demonstrated this in the other tutorials - so perhaps there is some value in telling users that it is possible to add tags...

One suggestion: Perhaps we can omit `GetTags()` and the addition of 'global' tags on the `ReportingConfiguration` while at the same time preserving tags on the scenario level (e.g. `"Sun"`)? If you do end up doing this, consider making similar change to add tag as part of the scenario in the other tutorials.